diffyscan/utils/calldata.py

```diff
@@ -0,0 +1,45 @@
+from .logger import logger
+from .encoder import encode_constructor_arguments
+
+def get_calldata(contract_address_from_config, target_compiled_contract, binary_config):
+    calldata = get_prepared_calldata_from_config(contract_address_from_config, binary_config)
+    if calldata is not None:
+          return calldata, ''
```

Comment:
```suggestion
          return calldata, None
```
Here, we have a function that returns `calldata` and error (optionally). 

Why return empty text instead of `None` in such a case?